File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l=[]
        for i in range(len(nums)):
            val=target-nums[i]
            if val in nums and nums.index(val)!=i:
                l.append(i)
                l.append(nums.index(val))
                return l
        return l

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> mp=new HashMap<>();
        int ans[]=new int[2];
        for(int i=0; i<nums.length; i++){
            int val=target-nums[i];
            if(mp.containsKey(val)){
                ans=new int[]{i,mp.get(val)};
                return ans;
            }
            mp.put(nums[i],i);
        }
        return ans;
    }
}